Packs/VMRay/Integrations/VMRay/VMRay.yml

```diff
@@ -6,7 +6,7 @@ configuration:
 - additionalinfo: Reliability of the source providing the intelligence data.
   defaultvalue: C - Fairly reliable
   display: Source Reliability
-  name: integrationReliability
+  name: reliability
```

Comment:
changing a param name is a breaking change, is this because of an SDK error?
If so, I suggest putting this param as `hidden: true` and adding a new one with the correct `name`.

In the code, you would have to handle the case where the old parameter name is used as well as the new one.